java/core/src/main/java/co/worklytics/psoxy/rules/google/PrebuiltSanitizerRules.java

```diff
@@ -37,12 +38,32 @@
             .build())
         .build();
 
+    static final Set<String> GOOGLE_CHAT_EVENT_PARAMETERS_PII = ImmutableSet.of(
+        "actor"
+    );
+    static final Set<String> GOOGLE_CHAT_EVENT_PARAMETERS_ALLOWED = ImmutableSet.<String>builder()
+        .addAll(GOOGLE_CHAT_EVENT_PARAMETERS_PII)
+        .add("room_id", "timestamp_ms", "message_id", "room_name")
+        .build();
+
+
     static final Rules GOOGLE_CHAT = Rules.builder()
         .allowedEndpointRegex("^/admin/reports/v1/activity/users/all/applications/chat.*")
         .pseudonymization(Rule.builder()
-            .relativeUrlRegex("/admin/reports/v1/activity/users/all/applications/chat.*")
+            .relativeUrlRegex("^/admin/reports/v1/activity/users/all/applications/chat.*")
             .jsonPath("$..email")
-            .jsonPath("$.items[*].events[*].parameters[?(@.name in ['actor'])].value")
+            .jsonPath("$.items[*].events[*].parameters[?(@.name in [" +
+                GOOGLE_CHAT_EVENT_PARAMETERS_PII.stream().map(s -> "'" + s + "'").collect(Collectors.joining(",")) +
+                "])].value")
+            .build()
+        )
+        .redaction(Rules.Rule.builder()
+            .relativeUrlRegex("^/admin/reports/v1/activity/users/all/applications/chat.*")
+            // this build a negated JsonPath predicate for all allowed event parameters, so anything other
+            // than expected headers will be redacted. Important to keep ".*$" at the end.
```

Comment:
The " Important to keep ".*$" at the end." is no longer needed. Was just for the negative look-ahead. Review all comments like this